tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -32,16 +35,52 @@ spec:
     - name: pipelineRunUid
       type: string
       description: The uid of the current pipelineRun. Used as a label value when creating internal requests
+    - name: taskGitUrl
+      type: string
+      description: The url to the git repo where the release-service-catalog tasks to be used are stored
+    - name: taskGitRevision
+      type: string
+      description: The revision in the taskGitUrl repo to be used
+    - name: pyxisServer
+      type: string
+      description: The server type to use. Options are 'production','production-internal,'stage-internal' and 'stage'.
+      default: production
+    - name: pyxisSecret
+      type: string
+      description: |
+        The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert
   workspaces:
     - name: data
       description: workspace to read and save files
   steps:
     - name: sign-image
-      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      image: quay.io/konflux-ci/release-service-utils:7d0135b80a47cdaa225010ea1e2dff78d057c922
+      computeResources:
+        limits:
+          memory: 4Gi
+          cpu: '2'
+        requests:
+          memory: 2Gi
+          cpu: '2'
+      env:
+        - name: pyxisCert
+          valueFrom:
+            secretKeyRef:
+              name: $(params.pyxisSecret)
+              key: cert
+        - name: pyxisKey
+          valueFrom:
+            secretKeyRef:
+              name: $(params.pyxisSecret)
+              key: key
       script: |
         #!/usr/bin/env bash
         #
-        set -ex
```

Comment:
Why remove the `-x`? (Related to Johnny's questions elsewhere.)

Reply:
I believe it was due to the sheer amount of logs that I was seeing and it causes log display issues. I can re-add and see.

Reply:
OK, I leave it to your judgment. If it's removed and we end up missing it, we can always add it back.

Reply:
reverted